onnxruntime/python/tools/quantization/onnx_quantizer.py

```diff
@@ -723,7 +723,21 @@
             or (tensor_name in self.generated_value_names)
         )
 
-    def quantize_inputs(
+    def quantize_activation(self, node, indices, from_subgraph=False):
```

Comment:
>quantize_weight [](http://example.com/codeflow?start=8&length=15)

Consider adding comments explaining: in what circumstances a weight is not an intializer?